### PR TITLE
Add enableSilentExposureLogging in ConsumableTestDefinition

### PR DESCRIPTION
**Pull request by bayareabear**
Add lastRevisionDate in TestDefinition so that we can log the test according to their update activity time

### CODE REVIEW

File: proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java [hunk 1]
```diff
@@ -55,6 +56,8 @@ public class TestDefinition {
 
     private boolean enableUnitlessAllocations;
 
+    private Date lastRevisionDate;
```
Comment (zacharygoodwin) [inline proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java:59]: One thing to consider. Using date object will take up more memory than if we use a the boolean and calculate in proctor-builder based on "lastRevisionDate - build date > X". Can discuss with the rest of team if worth doing it this way

Comment (bayareabear) [reply proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java]: agreed. boolean is better

File: proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java [hunk 2]
```diff
@@ -55,6 +55,8 @@ public class TestDefinition {
 
     private boolean enableUnitlessAllocations;
 
+    private boolean silentExposureLogging;
```
Comment (jamesallen-indeed) [inline proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java:58]: nit: Based on PROC-1413, should this be `isExposureLoggingSilent` or `enableSilentExposureLogging` or something else? (I'm terrible with names).

**Comment by zacharygoodwin** [comment]
Need to add field to the [ConsumableTestDefinition](https://github.com/indeedeng/proctor/blob/master/proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java).

May only need to update the [ConsumableTestDefinition](https://github.com/indeedeng/proctor/blob/master/proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java) and can leave the test definition class alone since the field will be populated by proctor builder rather than stored in the test definition entry in the DB. When the builder runs it calls:  https://github.com/indeedeng/proctor/blob/master/proctor-common/src/main/java/com/indeed/proctor/common/ProctorUtils.java#L158 which calls the fromTestDefintion to convert to the object proctor consumers use.

File: proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java (hunk 2 above)
Comment (bayareabear) [reply proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java]: updated

**Review by zacharygoodwin — APPROVED** [review]

**Comment by bayareabear** [comment]
This is no longer needed for now

**Pull request closed without merging**